README.md

```diff
@@ -67,3 +67,13 @@ get your code included into the project.
 3. Create new Pull Request
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2FREADME.png)
+
+## Tested Operating Systems and Node Versions
+
+Currently, the tests in this repository are running against:
```

Comment:
Should we call out this is for the client libraries? As the management ones build in a separate yml file

Reply:
specified client libraries